backend/app/api/diagnosis.py

```
@@ -31,6 +36,51 @@ class CreateChatMessageRequest(BaseModel):
+        image.save(file_path, "WEBP", quality=85)
+        
+        # 4. 접근 가능한 URL 반환
+        image_url = f"/uploads/diagnosis/{file_id}.webp"
+        return {"image_url": image_url}
+        
+    except Exception as e:
+        logger.exception("Image upload and processing failed")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"이미지 처리 중 오류가 발생했습니다: {str(e)}"
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)